src/main/java/org/isf/accounting/gui/PatientBillEdit.java

```diff
@@ -1057,58 +1133,51 @@ private JButton getJButtonSave() {
 			jButtonSave.setHorizontalAlignment(SwingConstants.LEFT);
 			jButtonSave.addActionListener(actionEvent -> {
 
-				if (listSelected == null) {
-					listSelected = lstArray.get(0);
+				if (thisBill.getPriceList() == null) { // FIXME: workaround ? to be removed ?
+					thisBill.setPriceList(lstArray.get(0));
 				}
 
-				if (this.insert) {
-					RememberDates.setLastBillDate(billDate);             //to remember for next INSERT
-					Bill newBill = new Bill(0,                        //Bill ID
-							billDate,                                    //from calendar
-							billDate,                                    //most recent payment
-							true,                                   //is a List?
-							listSelected,                                //List
-							listSelected.getName(),                      //List name
-							thisBill.isPatient(),                        //is a Patient?
-							thisBill.isPatient() ?
-									patientSelected : null,    //Patient ID
-							thisBill.isPatient() ?
-									patientSelected.getName() :
-									jTextFieldPatient.getText(),         //Patient Name
-							paid ? "C" : "O",                            //CLOSED or OPEN
-							total.doubleValue(),                         //Total
-							balance.doubleValue(),                       //Balance
-							user,                                       //User
-							thisBill.getAdmission());					//Admission
-									
+				if (insert) {
+					Bill newBill = new Bill(0, // Bill ID
+									thisBill.getDate(), // from calendar
```

Comment:
funny spacing/alignment

Reply:
I'm using auto-formatting upon save, so the spacing is the one decided by the ide formatter (I don't change anything)